Dockerfile

```diff
@@ -137,6 +137,8 @@ RUN export DEBIAN_FRONTEND=noninteractive && \
         libvips-tools \
         libyaml-dev \
         mercurial \
+        musl \
```

Comment:
can you add a comment to the two lines that it's for rust?

like:

```
# used for rust see issue #issueNumber
musl \ 
musl-tools \
```

Trying to get the use cases for tools (as we have probably a lot that we don't need)

Reply:
I guess dockerlint does not like my comment. 
https://github.com/RedCoolBeans/dockerlint/issues/54
https://github.com/RedCoolBeans/dockerlint/issues/73

Might make sense to look into using hadolint instead, as dockerlint seems unmaintained.

Reply:
@lukasholzer What should I do about this here?